http/src/main/resources/init-resources/rstudio-docker-compose.yaml

```diff
@@ -16,8 +16,10 @@ services:
       WORKSPACE_NAMESPACE: "${GOOGLE_PROJECT}"
       CLUSTER_NAME: "${CLUSTER_NAME}"
       OWNER_EMAIL: "${OWNER_EMAIL}"
+    privileged: true
```

Comment:
just for my education, what does this do?

Reply:
Hm, it gives the container deeper control of all VM devices. It was in the example docker-compose RStudio sent. I also see this:

https://github.com/rstudio/rstudio-docker-products/blob/3f22fbff6c35f185253b507746187a8f07023e1c/README.md#privileged-containers

I can see if things work without it, and also potentially ask RSP team if it's really necessary.

Also cc @adrazhi, do we have concerns about running docker containers in user VMs in `privileged` mode?

Reply:
I actually removed it for now

Reply:
Leaving `--privileged` it out of this PR, opened https://broadworkbench.atlassian.net/browse/IA-1868 for future investigation